Sources/Socket.swift

```diff
@@ -104,6 +104,37 @@ class Socket {
     }
   }
 
+  func bind(path: String) throws {
+    var addr = sockaddr_un()
+    addr.sun_family = sa_family_t(AF_UNIX)
+
+    // addr.sun_path is a `char [N]` and become a tuple in Swift.
+    // Its size is different on different platforms.
+    // Details can be found in `man 7 unix`
```

Comment:
This would rest in not being able to use the type alias below, but I think instead this could use [`Data`](https://github.com/kylef/Curassow/blob/master/Sources/Data.swift) allocated with the capacity.